bin/rspecq

```diff
@@ -154,6 +154,8 @@ else
   redis_opts[:host] = opts[:redis_host]
 end
 
+Sentry.init if ENV['SENTRY_DSN']
```

Comment:
Shouldn't that be:

```ruby
Sentry.init do |config|
  config.dsn = if ENV['SENTRY_DSN']
end
```

Reply:
I don't believe so; Sentry reads from `ENV['SENTRY_DSN']` by default (see below), so all that needs to happen is the call to `Sentry.init`:

https://github.com/getsentry/sentry-ruby/blob/b61bc0de2266b0abe76ad3bc5161e2da2a239bc5/sentry-ruby/lib/sentry/configuration.rb#L221